src/Files.Backend/ViewModels/Dialogs/FileSystemDialog/FileSystemDialogViewModel.cs

```diff
@@ -137,29 +137,21 @@ public static FileSystemDialogViewModel GetDialogViewModel(FileSystemDialogMode
 				// Subtitle text
 				if (conflictingItems.Count > 1)
 				{
-					if (nonConflictingItems.Count > 0)
-					{
-						// There are {0} conflicting file names, and {1} outgoing item(s)
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleMultipleConflictsMultipleNonConflicts".ToLocalized(), conflictingItems.Count, nonConflictingItems.Count);
-					}
-					else
-					{
-						// There are {0} conflicting file names
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleMultipleConflictsNoNonConflicts".ToLocalized(), conflictingItems.Count);
-					}
+					var descriptionLocalized = (nonConflictingItems.Count > 0)
+						? "ConflictingItemsDialogSubtitleMultipleConflictsMultipleNonConflicts".ToLocalized() // There are {0} conflicting file names, and {1} outgoing item(s)
+						: "ConflictingItemsDialogSubtitleMultipleConflictsNoNonConflicts".ToLocalized(); // There are {0} conflicting file names
+
+					descriptionText = string.Format(descriptionLocalized, conflictingItems.Count, nonConflictingItems.Count);
+				}
+				else if (nonConflictingItems.Count > 0)
+				{
+					// There is one conflicting file name, and {0} outgoing item(s)
+					descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictMultipleNonConflicts".ToLocalized(), nonConflictingItems.Count);
 				}
 				else
 				{
-					if (nonConflictingItems.Count > 0)
-					{
-						// There is one conflicting file name, and {0} outgoing item(s)
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictMultipleNonConflicts".ToLocalized(), nonConflictingItems.Count);
-					}
-					else
-					{
-						// There is one conflicting file name
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictNoNonConflicts".ToLocalized(), conflictingItems.Count);
-					}
+					// There is one conflicting file name
+					descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictNoNonConflicts".ToLocalized(), conflictingItems.Count);
 				}
```

Comment:
I think we should use the ternary operator here.
Something like:

![1](https://user-images.githubusercontent.com/102259289/199838266-313d4119-9618-4980-9940-bbeba92dff80.png)

Reply:
Done.